baseband/vdif/base.py

```diff
@@ -543,36 +549,39 @@ def write(self, data, invalid_data=False):
 
 
 open = make_opener('VDIF', globals(), doc="""
---- For reading : (see :class:`VDIFStreamReader`)
+--- For reading a stream : (see :class:`VDIFStreamReader`)
```

Comment:
Does this link work? Elsewhere, we spell out `~baseband.vdif.base.VDIFStreamReader`